.github/scripts/check_migrations.py

```diff
@@ -0,0 +1,54 @@
+import subprocess
+import sys
+
+
+def get_diff():
+    subprocess.run(
+        ["git", "fetch", "origin"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL
+    )
+    output = [
+        f"./{file_path}"
+        for file_path in subprocess.check_output(
+            ["git", "diff", "--name-only", "origin/develop"]
+        )
+        .decode()
+        .splitlines()
+    ]
+    if not output:
+        output = [
+            f"./{file_path}"
+            for file_path in subprocess.check_output(
+                ["git", "diff", "--name-only", "origin/main"]
+            )
+            .decode()
+            .splitlines()
+        ]
```

Comment:
Query - what it this actually doing?

Reply:
I think it's getting all the changed lines in the diff between this branch and the main branch. We can then use it to check for the keywords.

Reply:
What is the diff about?